### DISCUSSION TITLE
I heard Axios is no longer maintained and Inertia uses it - what's going to happen?

### DISCUSSION THREAD

**Discussion by stackerito** [inertiajs/inertia#858] [Help] [answered]
I saw that Axios is no longer maintained. Should I worry now that I want to start a project with Inertia?
What's going to happen?

**Comment by dinoluck** [comment, marked as answer]
Saw where? Afaik they are indeed supporting the project still.

**Reply by stackerito** [reply]
Thank you, some people on other forums told me and I also saw a few times in their discussions history on GitHub where people asked if the project is "dead" and they were answered by the developers that they don't have time to work on it. However that was from 2019. I guess now it's OK as I see many contributors right now, but wasn't sure

**Reply by jonagoldman** [reply]
it's not dead but also not needed anymore. I think is better to use the native apis in modern browsers.

**Reply by claudiodekker** [reply]
In addition to what's said above, we do eventually (pre-1.0) intend to remove Axios altogether in favour of XHR/Fetch. 

If you want to program with the future in mind, it's best to use those instead of Axios even though Inertia uses it internally. That way, once we do remove Axios, it'll instantly be removed from your own (compiled) .js bundles as well 👍

Of course, if you do want to use Axios in your own code-base, even after we remove it, then by all means go ahead and use it. There's nothing wrong with it per-se, we just feel like it's an unnecessary dependency as we use only a small portion of it's capabilities. We simply haven't gotten around to replacing/removing it just yet.

**Reply by craigrileyuk** [reply]
@claudiodekker - I was just about to ask about this since axios is a pretty hefty 27KB dependency and I didn't think it would've been used for much under the hood.
